doc/decisions/decision_process.md

```diff
@@ -3,14 +3,16 @@
 ## Problem
 
 Simply discussing in an issue and then implementing a solution is okay for non-substantial changes.
-Substantial decisions must be made in a transparent and participative way.
+Substantial decisions, however, must be made in a transparent and participative way.
+
+The main purpose of the decision process is to get a common understanding of the problems and the impacts of possible solutions.
 
 ## Constraints
 
 - All relevant information about decisions must be within Elektra's repository.
 - All decisions must go through at least two review rounds, with a merge in between.
-- At least two people need to be in favor of the decision in both rounds.
-- [Documentation guidelines](/doc/contrib/documentation.md) apply.
+- At least two people need to approve the decision in each round.
+- [Documentation guidelines](https://www.libelektra.org/devgettingstarted/documentation) apply.
```

Comment:
Small note, but I'd prefer if we use relative links, where possible. That way people reading on GitHub won't be directed to the website, if the prefer GitHub. It also means you can use the link locally in your editor and don't need to first find out what file this is actually referring to. 

```suggestion
- [Documentation guidelines](../contrib/documentation) apply.
```

Reply:
This should be also clarified in documentation guidelines itself, can you please do an update there? Btw. I dislike `..` going to the root, this shouldn't be encouraged.